### PR TITLE
python3.pkgs.telfhash: init at unstable-2021-01-29

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119991` at 08fbac79 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.telfhash</li>
    <li>python39Packages.telfhash</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.042s’
Near pkgs/development/python-modules/telfhash/default.nix:33:0:
```
   |
33 |     description = "Symbol hash for ELF files";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.041s’
Near pkgs/development/python-modules/telfhash/default.nix:33:0:
```
   |
33 |     description = "Symbol hash for ELF files";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/telfhash/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, capstone
+, nose
+, pyelftools
+, tlsh
+}:
+buildPythonPackage {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/telfhash/default.nix:9]:
```suggestion
}:

buildPythonPackage {
```

File: pkgs/development/python-modules/telfhash/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  src = fetchFromGitHub {
+    owner = "trendmicro";
+    repo = "telfhash";
+    rev = "b5e398e59dc25a56a28861751c1fccc74ef71617";
+    sha256 = "jNu6qm8Q/UyJVaCqwFOPX02xAR5DwvCK3PaH6Fvmakk=";
+  };
+
+  # The tlsh library's name is just "tlsh"
+  preBuild = ''
+    sed 's@python-tlsh@tlsh@' -i requirements.txt
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/telfhash/default.nix:22]: Please use substituteInPlace.

File: pkgs/development/python-modules/telfhash/default.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  '';
+
+  propagatedBuildInputs = [
+    capstone
+    nose
+    pyelftools
+    tlsh
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/telfhash/default.nix:31]: missing checkPhase and pythonImportsCheck.

**Pull request merged**